packages/react-components/src/molecules/Accordion.tsx

```diff
@@ -0,0 +1,116 @@
+import { FC, useState } from 'react';
+import { css } from '@emotion/react';
+import { Card, Headline5, Paragraph, Link } from '..';
+import { lineHeight, perRem } from '../pixels';
+import { chevronUpIcon, externalLinkIcon } from '../icons';
+import { isInternalLink } from '../utils';
+import { charcoal, silver, steel } from '../colors';
+
+const containerStyles = css({
+  padding: `0 ${9 / perRem}em`,
+});
+
+const itemStyles = css({
+  borderBottom: `${steel.rgb} solid 1px`,
+  '~ div:last-of-type': {
+    borderBottom: 'none',
+  },
+  padding: `${9 / perRem}em 0`,
+});
+
+const headerStyles = css({
+  display: 'grid',
+  width: '100%',
+  gridColumnGap: `${16 / perRem}em`,
+  gridTemplateColumns: 'min-content 1fr min-content',
+  padding: `${9 / perRem}em ${15 / perRem}em`,
+  cursor: 'pointer',
+
+  ':hover': {
+    background: silver.rgb,
+    borderRadius: '4px',
+  },
+});
+
+const iconStyles = css({
+  width: `${lineHeight / perRem}em`,
+  height: `${lineHeight / perRem}em`,
+  alignSelf: 'center',
+  svg: {
+    stroke: charcoal.rgb,
+    fill: charcoal.rgb,
+  },
+});
+
+const iconOpenStyles = css({
+  svg: {
+    transition: '250ms',
+
+    transform: 'rotate(180deg)',
+  },
+});
+const bodyStyles = css({
```

Comment:
is there a reason there isn't an empty line between these style variables? What about the extra line at line 48?

Reply:
Fixed